xarray/core/common.py

```diff
@@ -115,7 +120,15 @@ def __iter__(self):
     def T(self):
         return self.transpose()
 
-    def get_axis_num(self, dim):
+    @overload
+    def get_axis_num(self, dim: str) -> int:
+        pass
```

Comment:
I think it's more idiomatic to use `...` rather than `pass` when writing overloads?

Reply:
fixed